include/exadg/operators/inverse_mass_operator.h

```diff
@@ -94,6 +154,13 @@ class InverseMassOperator
   dealii::MatrixFree<dim, Number> const * matrix_free;
 
   unsigned int dof_index, quad_index;
+
+  bool explicit_matrix_free_inverse_mass_available;
+
+  MassOperator<dim, n_components, Number> mass_operator;
+
+  std::shared_ptr<BlockJacobiPreconditioner<MassOperator<dim, n_components, Number>>>
+    mass_preconditioner;
```

Comment:
In the context of this class (`InverseMassOperator`), the name `elementwise_inverse_mass` would be more intuitive than `mass_preconditioner`.